src/app/(proper_react)/redesign/(authenticated)/user/dashboard/View.tsx

```diff
@@ -118,28 +118,23 @@ export const View = (props: Props) => {
 
   // MNTOR-2226
   const getExposureCardLink = (exposure: Exposure) => {
-    let link = "/redesign/user/dashboard";
-
     if (!isScanResult(exposure)) {
       // if it's a data breach exposure
       if (props.isEligibleForFreeScan) {
-        link =
-          "redesign/user/dashboard/fix/data-broker-profiles/start-free-scan";
+        return "redesign/user/dashboard/fix/data-broker-profiles/start-free-scan";
```

Comment:
Semi-global to this fn, but do these paths need leading `/` for absolute URLs (similar to L137 below)?

Reply:
Thanks, fixed